### PR TITLE
kubernetes: Use the kube config server address

### PR DESCRIPTION
**Pull request by stefwalter**
Connect to the address specified in the .kube/config file. It
may be a VM or a docker container.

### CODE REVIEW

**Comment by sub-mod** [comment]
LGTM
with this change i am able to see the cluster

**Comment by stefwalter** [comment]
Added another commit to add the "validate" TLS option. This turns on certificate validation, now that we're possibly connecting off host.

We hook up the "validate" option to the ```~.kube/config``` "insecure-skip-tls-verify" option.

File: src/bridge/cockpitchannel.c [hunk 1]
```diff
@@ -977,6 +978,7 @@ cockpit_channel_parse_connectable (CockpitChannel *self,
   const gchar *internal;
   const gchar *address;
   JsonObject *options;
+  gboolean local = FALSE;
```
Comment (petervo) [inline src/bridge/cockpitchannel.c]: Shouldn't we default to TRUE and only set false when we have the address argument?

Comment (stefwalter) [inline src/bridge/cockpitchannel.c]: FALSE is the more secure alternative ... I would like to fail safe here.

**Comment by sub-mod** [comment]
screenshot from 2015-08-25 14-56-55

@stefwalter I am able to connect , but not sure if this introduced a bug in the UI.
I am able to see kubernetes service.

**Comment by stefwalter** [comment]
>  I am able to see kubernetes service.

That's a separate bug. I see this behavior on master. Can you file it separately?

**Pull request closed without merging**